temporalcli/commandsgen/commands.yml

```
@@ -3162,6 +3136,119 @@ commands:
         type: bool
         description: Don't prompt to confirm.
 
+  - name: temporal task-queue config
+    summary: Get and set Task Queue configuration
+    description: |
+      Manage Task Queue configuration:
+
+      ```
+      temporal task-queue config [command] [options]
+      ```
+
+      Available commands:
+      - `get`: Retrieve the current configuration for a task queue
+      - `set`: Update the configuration for a task queue
+
+  - name: temporal task-queue config get
+    summary: Get Task Queue configuration
+    description: |
+      Retrieve the current configuration for a Task Queue:
+
+      ```
+      temporal task-queue config get \
+          --task-queue YourTaskQueue \
+          --task-queue-type activity
+      ```
+
+      This command returns the current configuration including:
+      - Queue rate limit: The overall rate limit of the task queue.
+        This setting overrides the worker rate limit if set.
+        Unless modified, this is the system-defined rate limit.
+      - Fairness key rate limit defaults: Default rate limits for fairness keys.
+        If set, each individual fairness key will be limited to this rate,
+        scaled by the weight of the fairness key.
+
+    options:
+      - name: task-queue
+        type: string
+        description: |
+          Task Queue name.
+        required: true
+        short: t
+      - name: task-queue-type
+        type: string-enum
+        description: |
+          Task Queue type.
+          Accepted values: workflow, activity, nexus.
+        required: true
+        enum-values:
+          - workflow
+          - activity
+          - nexus
+
+  - name: temporal task-queue config set
+    summary: Set Task Queue configuration
+    description: |
+      Update a Task Queue's overall rate limit and the default rate limit for all fairness keys:
```

```suggestion
      Update configuration settings for a Task Queue.
```
I guess I missed this but like the get we can try to keep this generic to task queue configuration. Let's not limit this to the rate limit, we can add more things to this command later.